README.rst

```diff
@@ -93,11 +92,11 @@ This requires admin privileges to http://sympy-live.appspot.com. If you don't
 have access to this App Engine application, but want to test it, see the
 instructions in the `Testing on the App Engine`_ section below.
 
-Finally, go to http://NN.sympy-live.appspot.com, where ``NN`` is the version
-you just uploaded, and make sure that it works.  If it does, go to the
-``Versions`` section of the sympy-live dashboard, and set this as the new
-default version.  If there are any issues, you can roll back to the previous
-version from this same screen.
+You can open the newly deployed web app with::
+
+    $ gcloud app browse
+
+which will open https://sympy-live-hrd.appspot.com/ in a browser.
```

Comment:
It looks like you can access specific versions with `https://NN-dot-sympy-live-hrd.appspot.com/`. This URL is also available at the top of the dashboard if you select a version from the dropdown.